packages/enzyme-adapter-utils/src/RootFinder.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+/**
+ * This component is a utility to assist in the finding of the root component
+ * when the component is mounted in a wrappingComponent.
+ */
+export default class RootFinder extends React.Component {
+  render() {
+    const { children } = this.props;
+
+    return children;
+  }
+}
+RootFinder.propTypes = {
+  children: PropTypes.node.isRequired,
+};
```

Comment:
This component is only ever used in `createMountWrapper`, so I think the code would actually be a bit easier to understand if it just lived in that file. But eslint forbids that so I stuck it here. Do you think this could warrant a rule override?